client/src/views/home/OrganizationListPage.vue

```diff
@@ -105,51 +105,11 @@
               class="organization-list-row__col"
               size="3"
             >
-              <ion-card
-                button
-                class="organization-card"
+              <organization-card
+                :device="device"
+                :last-login-device="storedDeviceDataDict[device.deviceId]"
```

Comment:
Use the optional chaining operator to pass the `lastLogin` directly:
`:last-login-device="storedDeviceDataDict[device.deviceId]?.lastLogin"`